### ISSUE TITLE
Metadata in Shapes

### ISSUE THREAD

**Issue by sebbader** [International-Data-Spaces-Association/InformationModel#281]
A lot of our Shapes have ontology metadata, for instance at: https://github.com/International-Data-Spaces-Association/InformationModel/blob/33426b25cf6074c6513af48b34550600930dff8e/testing/contract/ActionShape.ttl#L11

Do we really need those? I just came up with an approach to collect all Shapes through owl:imports in one TTL file, similar to the Ontology.ttl appraoch we already have. However, Protege has troubles collecting the Shapes caused by these metadata statements.

**Comment by JohannesLipp** [comment]
Good point indeed! This is only required if the same shapes also use SPARQL queries (cf. https://www.w3.org/TR/shacl/#sparql-prefixes). Task = Remove all `sh:declare` statements that are not used in SPARQL queries.
